### PR TITLE
fixed listingBookings insert, added util for dateRange to isoString, …

### PR DESCRIPTION
**Pull request by joe-nm**
…added tests for datesToISOString

### CODE REVIEW

File: db/coms.js [hunk 1]
```diff
@@ -47,21 +51,19 @@ Listings = function(input={}) {
   this.availability = input.availability;
   this.host_id = input.host_id;
 };
-
+  
 Experiences = function(input={}) {
```
Comment (messerc) [inline db/coms.js]: Add variable declarations here - `const` for example is a great new way to declare variables. It's part of Emcascript 2015 (you might see it as "ES6" online), and it's sweeping over the Javascript world. `let` is another, but that is generally to indicate a variable that mutates.

File: server.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const PORT = process.env.PORT || 8000;
 const coms = require('./db/coms.js');
 coms.client.connect();
 
-const {convertArrayToPG} = require('./utils/pgFormat.js');
+const pgFormat = require('./utils/pgFormat.js');
```
Comment (messerc) [inline server.js]: De-structure this - stay consistent with line 11

File: db/coms.js (hunk 1 above)
Comment (tcparker976) [reply db/coms.js]: I agree with Chris. I would suggest reviewing ES6 syntax --> it could really help you out here.

Comment (tcparker976) [reply db/coms.js]: Really solid code Joe. For minor feedback I would say I think some of your variable declarations could be a bit more clear. I'm very impressed with your 47 tests coverage.

**Pull request merged**